Sources/Basics/FileSystem/FileSystem+Extensions.swift

```diff
@@ -465,7 +485,7 @@ extension FileSystem {
             return try swiftSDKsDirectory
         }
     }
-
+    
```

Comment:
nit: unnecessary trailing newline. To get rid of these altogether you can run `./Utilities/soundness.sh` in this package, assuming you have [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) installed.

Reply:
Resolving locally

Reply:
would you be able to push that formatting fix to this branch as well?

Reply:
still present in the diff